File: simulate.jl
using DelimitedFiles
using Plots

data = readdlm("initial_data.txt")
coordinates = data[:, 1:3]
speed = data[:, 4:end]
scatter3d(coordinates[:, 1], coordinates[:, 2], coordinates[:, 3],
          markersize = 2,
          markercolor = :red,
          xlabel = "X Label",
          ylabel = "Y Label",
          zlabel = "Z Label",
          legend = false)

File: simulate.py
import numpy as np
import matplotlib.pyplot as plt

data = np.loadtxt('initial_data.txt')
coordinates = data[:, :3]
speed = data[:, 3:] 
fig = plt.figure()
ax = fig.add_subplot(111, projection='3d')
ax.scatter(coordinates[:,0], coordinates[:,1], coordinates[:,2], c='r', marker='o')
ax.set_xlabel('X Label')
ax.set_ylabel('Y Label')
ax.set_zlabel('Z Label')
plt.show()
